docs/workflow/initialization.rst

```diff
@@ -262,6 +262,18 @@ If a particular code (let's say inspiral) wants to use an option supplied in the
   [inspiral-v1]
   channel-name = ${workflow|v1-channel}
 
+We also have access to anything specified in the environment when the workflow
```

Comment:
Rewrite with less repetition: 
> We also have access to all environment variables present when generating the workflow. These are accessed in the configuration file like this::

Reply:
maybe we should mention the special character exceptions .. 'to environment variables present when generating the workflow (with some exceptions). ..'